integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -285,8 +284,14 @@ void testCreateDomainUsingMultipleAuxiliaryImages() {
     createDomainAndVerify(domainUid, domainCR, domainNamespace,
         adminServerPodName, managedServerPrefix, replicaCount);
 
+    //create router for admin service on OKD
+    if (adminSvcExtHost == null) {
```

Comment:
Which method sets the variable adminSvcExtHost ? Is it a global ?

Reply:
adminSvcExtHost is a member variable. In this class there are tests reusing the previously created domain in the same domain space. In that case we don't need to recreate adminSvcExtHost unless it is null.
But specifically in this line we don't have to check if adminSvcExtHost is null because it is the first test and adminSvcExtHost has not been cerated before. Leave it there and it won't hurt.